sql/2019/04_Media/04_13.sql

```diff
@@ -0,0 +1,67 @@
+/*
+standard SQL
+04_06
+04_08
+04_13
+
+10 TB query
+only tested on the 1k mobile sample dataset
+
+this query goes into the response bodies and looks for the img tag where it has srcset inside the tag.
+I then pull out the image tags from each response and flatten them out of the array into strings
+I extract the srcset tag and the sizes tag from each img, and determine how many sources are present 
+(I am counting commas and then adding one to get the total, so commas in the url will screw with the numbers)
+the ratio is simply sources/sizes.
+
+I then break it into percentiles to see the breakdown of source images and sizes and ratios.  
+I also supply the count of scrset and size tags in the dataset.
+
+04_08
+I added a query for the src tag... and then feed theat up through the same queries for 04_06. 
+in the 1k mobile sample set 2918 have srcset, and 2932 have a fallback src
+
+04_13
+added in alt tag query.  get count and length - note that length 6 = alt="" (empty)
```

Comment:
Would the alt text Lighthouse a11y audit suffice? (Anything to avoid another response_bodies query)

Reply:
I remember looking at the lighthouse output, and going this way.. I dont recall why.

Reply:
One thing that is interesting in this query, is I can see how many of the alt texts are 6 characters alt="" - meaning that the system is all set up... but user failure.

If you think switching to the other is better, I can